src/main/java/io/gingersnapproject/cdc/EngineWrapper.java

```diff
@@ -118,19 +118,19 @@ public void taskStopped() {
    public void stop() throws IOException {
       engine.close();
       engine = null;
-      cacheService.stop(config.cache().uri());
+      cacheService.stop();
    }
 
    public void notifyError(Throwable t) {
       eventing.connectorFailed(name, t);
    }
 
    public void shutdownCacheService() {
```

Comment:
I don't think this stuff makes sense anymore either if we have a single configured cache service that we are bound to. If anything it should be controlled solely by the config watcher / rest endpoints to change the URI right?

Reply:
I don't think we need to support changing the URI at runtime at all tbh. If the ServiceBinding is updated then so is the pod.

Reply:
Sounds good to me.